conf/messages.en

```diff
@@ -1,7 +1,7 @@
 checkYourAnswers.title = Check your answers
 checkYourAnswers.heading = Check your answers
-checkYourAnswers.title.claimExpenses = Check your answers before claiming your expenses
-checkYourAnswers.heading.claimExpenses = Check your answers before claiming your expenses
```

Comment:
"Check your answers" is now being duplicated 4 times here, would it not be worth refactoring this down?

Reply:
done